# timestamp
Header header

# position and displacement of markers
float32[] marker_x
float32[] marker_y
float32[] marker_displacement_x
float32[] marker_displacement_y
